### ISSUE TITLE
Change nvidia stable driver to use production instead of latest

### ISSUE THREAD

**Issue by NklsCh** [NixOS/nixpkgs#365454]
## Describe the bug

Currently when you install the stable nvidia driver (system is NOT legacy) it installes the latest driver:
https://github.com/NixOS/nixpkgs/blob/3566ab7246670a43abd2ffa913cc62dad9cdf7d5/pkgs/os-specific/linux/nvidia-x11/default.nix#L40

As the Policy above states: https://github.com/NixOS/nixpkgs/blob/3566ab7246670a43abd2ffa913cc62dad9cdf7d5/pkgs/os-specific/linux/nvidia-x11/default.nix#L39 only the highest stable version should be used. <br>
Now to the Problem:
According to NVIDIA themself the driver listed as latest in the nixpkgs repo is actually bleeding edge and everything but stable:

![image](https://github.com/user-attachments/assets/8a7b51ef-860f-424f-b007-d1c7e9005819)

https://github.com/NixOS/nixpkgs/blob/3566ab7246670a43abd2ffa913cc62dad9cdf7d5/pkgs/os-specific/linux/nvidia-x11/default.nix#L51-L58

So basically could we just switch the line in https://github.com/NixOS/nixpkgs/blob/3566ab7246670a43abd2ffa913cc62dad9cdf7d5/pkgs/os-specific/linux/nvidia-x11/default.nix#L40
to 
`stable = if stdenv.hostPlatform.system == "i686-linux" then legacy_390 else production` which would install the actual stable driver

## Steps To Reproduce

1. ...
2. ...
3. ...

## Notify maintainers

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc
